packages/webview_flutter/webview_flutter_android/lib/src/android_webview.dart

```diff
@@ -1121,6 +1133,14 @@ class WebChromeClient extends JavaObject {
   /// mode.
   final HideCustomViewCallback? onHideCustomView;
 
+  final Future<void> Function(String message)? onHandleJavaScriptAlert;
+  final Future<bool> Function(String message)? onHandleJavaScriptConfirm;
+  final Future<String> Function(String message, String defaultValue)? onHandleJavaScriptPrompt;
+
+  final bool Function()? hasJavaScriptAlertCallback;
+  final bool Function()? hasJavaScriptConfirmCallback;
+  final bool Function()? hasJavaScriptPromptCallback;
```

Comment:
These can be removed. I don't believe they are used and they are redundant with the callback methods above.